styles.css

```diff
@@ -1,7 +1,7 @@
 :root {
   --color-white: #f3f3f3;
-  --color-darkblue: #1b1b32;
-  --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+  --color-darkblue: #370101;
+  --color-darkblue-alpha: rgb(28 9 9 / 59%);
```

Comment:
If your colors used are shades of brown, I think it would be better to use related names in the classes, instead of using dark blue.